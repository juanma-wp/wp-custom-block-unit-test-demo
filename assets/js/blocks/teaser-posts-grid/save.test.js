import React from 'react';
import renderer from 'react-test-renderer';
import Save from './save';

const mockSaveProps = {
	attributes: {
		items: [],
	},
};

describe( 'Teaser Posts Grid Block Save Component', () => {
	test( 'snapshot renders with defaults', () => {
		const component = renderer.create( <Save { ...mockSaveProps } /> );
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	test( 'snapshot renders with multiple items', () => {
		const items = [
			{ id: 13 },
			{ id: 42 },
		];
		const component = renderer.create( <Save { ...mockSaveProps } items={ items }/> );
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
