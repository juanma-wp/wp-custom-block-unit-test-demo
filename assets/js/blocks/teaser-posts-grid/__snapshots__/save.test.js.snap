// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teaser Posts Grid Block Save Component snapshot renders with defaults 1`] = `<ul />`;

exports[`Teaser Posts Grid Block Save Component snapshot renders with multiple items 1`] = `<ul />`;
